routers/routes/routes.go

```diff
@@ -385,24 +385,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Post("/keys/delete", userSetting.DeleteKey)
 		m.Get("/organization", userSetting.Organization)
 		m.Get("/repos", userSetting.Repos)
-
```

Comment:
These are unrelated.

Reply:
@lunny so should i add this in a second PR?

Reply:
Yes another PR and reference the PR that moved them too.

I suspect I merged it but if we're going about moving endpoints - the pr that does the movement should put the redirects in. (As fun as 2ndry bug fix PRs are, and we've had a flurry of them recently - mostly my fault - we should endeavour to get this right first time.)

Reply:
@lunny @zacheryph moved: #9538